### ISSUE TITLE
Control anchoring and docking

### ISSUE THREAD

**Issue by mansellan** [WaynePhillipsEA/twinbasic#44]
**Is your feature request related to a problem?**
A tiresome aspect of VB6 forms development is the boilerplate needed to adjust control sizes and locations when their containers resize. In the WinForms world (in .Net), controls can be either anchored or docked inside containers, which causes resizing and relocation to happen automatically.

**Describe the solution you'd like**
Anchoring and docking, similarly to WinForms

**Describe alternatives you've considered**
None

**Additional context**
Not sure how this would integrate with OCX controls - presumably code would have to be auto-generated to manipulate size and location properties.

**Comment by WaynePhillipsEA** [comment]
Absolutely;   this is one thing that really irks me with VB6 forms.  Well, one of many.
